pkg/reconciler/events/cache/cache.go

```diff
@@ -31,19 +31,6 @@ type eventData struct {
 	Run *v1alpha1.Run `json:"run,omitempty"`
 }
 
-// AddEventSentToCache adds the particular object to cache marking it as sent
-func AddEventSentToCache(cacheClient *lru.Cache, event *cloudevents.Event) error {
-	if cacheClient == nil {
-		return errors.New("cache client is nil")
-	}
-	eventKey, err := EventKey(event)
-	if err != nil {
-		return err
-	}
-	cacheClient.Add(eventKey, nil)
-	return nil
-}
-
 // IsCloudEventSent checks if the event exists in the cache
 func IsCloudEventSent(cacheClient *lru.Cache, event *cloudevents.Event) (bool, error) {
```

Comment:
looks like we need to update this function name and description 

```suggestion
// ContainsOrAddCloudEvent checks if the event exists in the cache, and if not, adds the event
func ContainsOrAddCloudEvent(cacheClient *lru.Cache, event *cloudevents.Event) (bool, error) {

Reply:
Whoops! Doing so now.